specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -998,6 +1005,13 @@
           "type": "array",
           "items": {
             "$ref": "#/definitions/AcsChatThreadParticipantProperties"
+          },
+          "metadata": {
```

Comment:
This is nested inside of the participants property which is causing a failure to generate the SDKs.

Reply:
There are participant metadata properties and thread metadata properties. Where should the participant metadata properties be nested so the SDKs generate?

Reply:
Are you saying that Metadata should not be a top-level property on AcsChatThreadCreatedEventData? This is different than all of the other events with Metadata.

Reply:
Participants is already defined as an array - we can't just turn that into an object as that is a breaking change.